.github/workflows/log-viewer-docs.yaml

```diff
@@ -0,0 +1,58 @@
+name: "docs"
+
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily at 00:15 UTC (the 15 is to avoid periods of high load)
+    - cron: "15 0 * * *"
```

Comment:
sorry for missing this previously - what's the reason we want to rebuild the docs on a schedule?